src/organisation/components/pba-numbers-form/pba-numbers-form.component.spec.ts

```diff
@@ -58,7 +58,7 @@ describe('PbaNumbersFormComponent', () => {
   let fixture: ComponentFixture<PbaNumbersFormComponent>;
 
   beforeEach(() => {
-    pipeSpy = spyOn(storeMock, 'pipe').and.returnValue(of(mockOrganisationDetails));
+    pipeSpy = spyOn(storeMock, 'pipe');
```

Comment:
Looks like this was previously intending to actually define some specific behaviour - was it just not, since we haven't needed to make changes elsewhere?

Reply:
With the upgrade if causing an error, which will explain why the previous dev removed it.

Reply:
Code updated